src/ocp_postprocess.rs

```diff
@@ -61,7 +62,16 @@ pub(crate) async fn ocp_postprocess(
     }
 
     if let Some(ip) = ip {
-        ip_rename(in_memory_etcd_client, ip, static_dirs, static_files)
+        // OpenShift uses `[ ]` to enclose IPv6 IPs also in non-URL fields. Let us add the brackets
```

Comment:
Is it truly correct for all places where we replace the IP?

Reply:
That's a great question, it seems to be that way, I'll put a list here so that you can review the location as well.

Reply:
This is what we have in etcd for an IPv6 IP with brackets:

```shell
/kubernetes.io/configmaps/openshift-apiserver/config
/kubernetes.io/configmaps/openshift-kube-apiserver/config-7
/kubernetes.io/configmaps/openshift-kube-apiserver/config-9
/kubernetes.io/configmaps/openshift-kube-apiserver/config-6
/kubernetes.io/configmaps/openshift-kube-apiserver/config
/kubernetes.io/configmaps/openshift-kube-apiserver/config-10
/kubernetes.io/configmaps/openshift-kube-apiserver/config-8
/kubernetes.io/configmaps/openshift-etcd/etcd-pod-3
/kubernetes.io/configmaps/openshift-etcd/etcd-scripts
/kubernetes.io/configmaps/openshift-etcd/etcd-pod-2
/kubernetes.io/configmaps/openshift-etcd/etcd-pod
/kubernetes.io/configmaps/openshift-etcd/restore-etcd-pod
/kubernetes.io/deployments/openshift-oauth-apiserver/apiserver
/kubernetes.io/operator.openshift.io/openshiftapiservers/cluster
/kubernetes.io/operator.openshift.io/authentications/cluster
/kubernetes.io/operator.openshift.io/kubeapiservers/cluster
/kubernetes.io/pods/openshift-kube-controller-manager/kube-controller-manager-another-hostname
/kubernetes.io/pods/openshift-oauth-apiserver/apiserver-6889868855-nhkqz
/kubernetes.io/pods/openshift-etcd/etcd-another-hostname
/kubernetes.io/replicasets/openshift-oauth-apiserver/apiserver-5c6d74c9db
/kubernetes.io/replicasets/openshift-oauth-apiserver/apiserver-6889868855
```

I believe we cover only those in recert currently. Wherever else I found the IP without brackets is not in static pods or in resources in etcd we should care (famous last words :) ).